tutorials/analysis/gMCS/tutorial_gMCS.m

```diff
@@ -0,0 +1,227 @@
+%% *genetic Minimal Cut Sets - gMCSs*
+%% Authors: I�igo Apaolaza, University of Navarra, TECNUN School of Engineering, Spain
+%% Luis V. Valcarcel, University of Navarra, TECNUN School of Engineering, Spain
+%% Francisco J. Planes, University of Navarra, TECNUN School of Engineering, Spain
+%% Reviewer(s): 
+%% INTRODUCTION
+% Minimal Cut Sets (MCSs) are minimal sets of reaction knockouts which deprive 
+% a network from accomplishing a given metabolic task$$^1$. On the other hand, 
+% genetic Minimal Cut Sets consist of minimal sets of genes whose simultaneous 
+% inhibition would render the functioning of a given metabolic task impossible$$ 
+% ^2$. Therefore, the concepts of MCSs and gMCSs are equivalent but at different 
+% levels: at the reaction level for the former and at the gene level for the latter.
+% 
+% MCSs are not directly converted into feasible knockout strategies at a 
+% gene level due to the fact that the Gene-Protein-Reaction (GPR) rules, typically 
+% provided in genome-scale metabolic reconstructions, are not one-to-one. Certainly, 
+% we can find impractical reaction knockouts because they are catalyzed by enzymes 
+% involving several genes, which increase the cost of the intervention. On the 
+% other hand, the resulting gene knockout interventions from MCSs may provoke 
+% undesired metabolic effects, as we may have reactions (included in a particular 
+% MCS) catalyzed by an enzyme that additionally participate in other relevant 
+% reactions (not included in the MCS under consideration). This fact is further 
+% discussed in Apaolaza et al., 2017(b)$$^3$.
+% 
+% Moreover, state-of-the-art methods that search for efficient gene knockout 
+% interventions use a combinatorial strategy  and, as a consequence, are not viable 
+% for seeking for high order solutions in large metabolic networks. This tutorial 
+% aims also to demonstrate this is possible with the function provided.
+%% EQUIPMENT SETUP
+%% Initialize The Cobra Toolbox and select the solver (~20 sec)
+% If necessary, initialise the cobre toolbox:
+%%
+initCobraToolbox
+%% 
+% Select the appropriate solver removing the "%" symbol only for the desired 
+% solver. Note that the approaches to search for MCS and gMCS problems are based 
+% on Mixed Integer Linear Programming (MILP).
+
+changeCobraSolver('ibm_cplex', 'all');
+% changeCobraSolver('gurobi', 'all');
+%% PROCEDURE
+% This tutorial will be divided into two different parts. First, a toy example 
+% will be used to illustrate the difference between MCSs and gMCSs and, second, 
+% gMCSs will be calculated for Recon2.v04 metabolic model$$^4$.
+%% 1. Toy example (10 sec ~ 1 min)
+% The toy example under study is presented below:
+% 
+% 
+% 
+% First, we are going to load the MAT-file which contains the metabolic network 
+% in the toy example.
+%%
+load('gMCStoyExample.mat');
```

Comment:
please use readCbModel instead of load.
This is to ensure that the models used are definitely compatible with the toolbox, and thus that the functions called are. 
Please also make sure, that this is also updated in the mlx file.